<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LR</class>
 <widget class="QWidget" name="LR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="te_contents">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>511</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_check">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>좌회전 우회전 체크</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>안녕하세요 저는 '좌우'입니다.
무엇을 도와드릴까요?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>리셋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bobae">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>보배드림</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mlb">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>500</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>mlb</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bbom">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>뽐뿌 정치게시판</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>570</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Version 0.0.4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
